ctru-rs/src/applets/swkbd.rs

```diff
@@ -7,6 +7,7 @@ use std::iter::once;
 use std::str;
 
 /// An instance of the software keyboard.
+#[derive(Clone)]
```

Comment:
does cloning the inner Box here end up cloning all the global state too? I would guess normally we don't want `Clone` for these "global-ish" kind of structs...